zastosowania-komputerow-w-elektrotechnice.csl

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Zastosowania Komputerów w Elektrotechnice</title>
+    <title-short>ZKwE</title-short>
+    <id>http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice</id>
+    <link href="http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice" rel="self"/>
+    <link href="http://www.zotero.org/styles/ieee" rel="template"/>
+    <link href="http://www.iee.put.poznan.pl/caiee/redakcja.doc" rel="documentation"/>
+    <author>
+      <name>Mateusz Pacek</name>
+      <email>mateusz.m.pacek@student.put.poznan.pl</email>
+    </author>
+    <contributor>
+      <name>Michael Berkowitz</name>
+      <email>mberkowi@gmu.edu</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <updated>2015-04-13T10:05:52+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="pl-PL">
```

Comment:
and then change the language here to "en-US" so these changes to terms will actually get picked up.